exonum/src/blockchain/mod.rs

```diff
@@ -449,6 +449,15 @@ impl BlockchainMut {
             .expect("Cannot update transaction pool");
     }
 
+    /// Performs several shallow checks that transaction is correct.
```

Comment:
```suggestion
    /// Performs that the transaction corresponds to existing instance of the service.
```
???

Reply:
As it said in the body of the `Dispatcher::check_tx`: "Currently the only check is that destination service exists, but later functionality of this method can be extended.".

Since this method just redirects a request, I believe it shouldn't assume exact checks that will happen.

And regarding the same doc-comment on the `Dispatcher::check_tx` -- I believe it's better to update its description once it will be stabilized.

Reply:
Ok. But there is no guarantee that the behaviour will be changed in the future. 🙃

Reply:
At least task in Jira proposes it. Anyway, it worth nothing to update description (and the name of the function) to match the nature of the function later, if we'll decide to leave things as-is.